flake.nix

```diff
@@ -131,6 +131,27 @@
                   Required when httpBootUrlTemplate uses HTTPS and the server certificate is not signed by a well-known CA.
                 '';
               };
+              netbootDirectory = lib.mkOption {
+                type = lib.types.nullOr lib.types.path;
+                default = "/netboot/mac";
```

Comment:
I'd rather this be null by default, this is a sorta "weird" place, and 2x weird to make it default